src/tiled/tiled.qbs

```diff
@@ -131,6 +131,8 @@ QtGuiApplication {
         "bucketfilltool.h",
         "capturestamphelper.cpp",
         "capturestamphelper.h",
+        "changecomponents.cpp",
+        "changecomponents.h",
```

Comment:
New files should also be added to the `src/tiled/tiled.pro` for now, until we can finally drop the qmake project.